packages/runtime-core/__tests__/hydration.spec.ts

```diff
@@ -18,10 +18,14 @@ import {
   createVNode,
   withDirectives,
   vModelCheckbox,
-  renderSlot
+  renderSlot,
+  Transition,
+  createCommentVNode,
+  vShow
 } from '@vue/runtime-dom'
 import { renderToString, SSRContext } from '@vue/server-renderer'
-import { PatchFlags } from '../../shared/src'
+import { PatchFlags } from '@vue/shared'
+import { vShowOldKey } from '../../runtime-dom/src/directives/vShow'
```

Comment:
@edison1105 for something like `vShowOldKey` that is needed in tests but should not be exposed as a public API, we can import them using relative path to the source file.